src/base-styles/helperClasses/animations/transitions.css

```diff
@@ -0,0 +1,18 @@
+/*---
+title: Transitions
+name: transitions
+section: Animation Helpers
+---
+
+Adds standard transitions
+
+class          | description
+-------------- | ---------------------------------------------------------
+`trsn-default` | transitions all properties to default duration and timing
+*/
+
+.trsn-default {
+  transition-property: all;
```

Comment:
`transition-property` has an initial value of `all`; we don't need to set it here

Reply:
It doesn't work for some reason if I don't put it (Chrome)